odoo/addons/base/models/qweb.py

```diff
@@ -1105,13 +1105,7 @@ def _compile_directive_out(self, el, options, indent):
         else:
             code.append(self._indent("force_display = None", indent))
 
-            if ttype == 't-esc':
-                # deprecated use.
-                code.append(self._indent(dedent("""
-                    if content is not None and content is not False:
-                        content = str(content)
-                """), indent))
-            elif ttype == 't-raw':
+            if ttype == 't-raw':
```

Comment:
same thing, is there a test to cover and explain this change?

Reply:
This actually restores the behaviour I'd decided upon when I merged the Markdown changes (because I didn't see the point of forcing escaping on t-esc, aside from ultimately making its replacement more difficult over the long term).